transcom/mymove
# INT B-18552 deprecate prime v1 createMTOShipment endpoint

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18552)

## Summary

We are deprecating the `v1` version of the Prime endpoint `createMTOShipment` per our [documentation](https://transcom.github.io/mymove-docs/api/prime/v1#tag/mtoShipment/operation/createMTOShipment).

I decided to leave the description for guidance on which endpoint that should be used, but removed all code and functionality. It was confirmed in the `proj-ghc` channel in Slack that HomeSafe does not use this endpoint any more. This is also documented in the Agility ticket.

### How to test

1. Fire up your environment
2. Hit up the url [for the Prime v1 Swagger UI](http://officelocal:3000/swagger-ui/prime.html#/)
3. Confirm `createMTOShipment` is deprecated
4. Attempt to use the endpoint via Swagger and should get a "not found" error

## Screenshots
![Screenshot 2024-08-26 at 3 25 02 PM](https://github.com/user-attachments/assets/412bda63-7e64-4026-bb84-2a17c1e4780f)
